docs/cloud/pouta/persistent-volumes.md

```diff
@@ -183,3 +183,33 @@ colleague to whom you want to transfer the volume.
 Your colleague can accept the transfer request of this volume:
 
     openstack volume transfer request accept <transferID> <authKey>
+
+## Expanding size to the attached volumes in the Pouta web interface
+
+Previuosly you have created and attached a volume. In this section you are going to enlarge the size of the volume attached to the instance. Before you attempt for volume expansion you have to detach the volume from the instance, please remember to unmount the volume before detaching it!
+
+    sudo umount /dev/vdb
+
+To expand the volume, first select the *Volumes* view in the Pouta web interface. Click the arrow symbol next to the **Edit Volume** buttonn for the volume you want to enlarge and select **Extend Volume**. Input the the volume amount you want to enlarge in (GiB) in the field **New Size (GiB)**. Finally, click the **Extend Volume** button.
+To attach an expanded volume similar to the previous attach persisitent volume, first select the *Volumes* view in the Pouta web interface. Click the arrow symbol next to the **Edit Volume** button for the volume you expanded and select **Manage attachments**. Select the instance (i.e. virtual machine) you want to attach the volume to in the **Attach to Instance** selector.
+
+![Expand persistent volume](/img/volume-expand-horizon1.png)
+
+Once you have logged in to your virtual machine, you can list the
+volumes:
+
+    sudo parted -l
+
+Similar to the previous persisitent volume creation you can identify the volume based on its size. First mount the path:
+
+    sudo mount /dev/vdb /media/volume
+
+Now you can check the status of the volume in a mounted XFS filesystem using the following command:
```

Comment:
Please check that you need to mount the volume before expanding it. It should be safer to expand an unmounted volume.

Reply:
yes we need it.

Reply:
`[cloud-user@tahir-jemal-volume-test ~]$  sudo xfs_growfs /dev/vdc
xfs_growfs: /dev/vdc is not a mounted XFS filesystem
[cloud-user@tahir-jemal-volume-test ~]$ sudo mount /dev/vdc /media/volume
[cloud-user@tahir-jemal-volume-test ~]$  sudo xfs_growfs /dev/vdc
meta-data=/dev/vdc               isize=512    agcount=20, agsize=655360 blks
         =                       sectsz=512   attr=2, projid32bit=1
         =                       crc=1        finobt=0 spinodes=0
data     =                       bsize=4096   blocks=13107200, imaxpct=25
         =                       sunit=0      swidth=0 blks
naming   =version 2              bsize=4096   ascii-ci=0 ftype=1
log      =internal               bsize=4096   blocks=2560, version=2
         =                       sectsz=512   sunit=0 blks, lazy-count=1
realtime =none                   extsz=4096   blocks=0, rtextents=0
data blocks changed from 13107200 to 26214400
[cloud-user@tahir-jemal-volume-test ~]$ 
`